plugins/ltac/g_tactic.mlg

```diff
@@ -59,6 +59,7 @@ open Extraargs
 
 (* idem for (x1..xn:t) [n^2 complexity but exceptional use] *)
 let check_for_coloneq =
+  let module LStream = Gramlib.LStream in
```

Comment:
Not sure this introduces a zero-cost module alias as in toplevel `module LStream = Gramlib.LStream`